### ISSUE TITLE
Urdu test set has data issues - tags which don't exist

### ISSUE THREAD

**Issue by AngledLuffa** [ltrc/IL-NER#5]
The Urdu training set has:

```
NEAR NEL NEN NEO NEP NETI
```

The test set has the following tags which never showed up in the training set:

```
['B-NET', 'B-NIMI', 'E-NET', 'E-NIMI', 'I-NET', 'I-NIMI']
```

Seeing as how NET and NIMI only occur a small number of times, and only in UR, I believe these to also be data bugs

**Comment by Pruthwik** [comment]
Updated the dataset and removed the noise. Thanks for pointing it out.

**Comment by AngledLuffa** [comment]
Thank you, I see you made most of these updates.

I did notice that `NEU` was one of the tags you'd updated, although it seems to have usable meaning, URL (or email address)

It was `NET` which occurs only once in the Urdu test data, in this sentence.  Here it seems to be an Organization, so I think the tag should be updated accordingly instead of just set to O:

```
حکمرانی O
اسیسمنٹ O
(       O
این     B-NET
ای      I-NET
ایس     I-NET
ڈی      I-NET
اے      I-NET
)       O
شروع    O
کیا     O
تاکہ    O
مرکز    O
سے      O
لے      O
کر      O
علاقائی  O
سطح     O
تک      O
مختلف   O
سرکاری  O
شعبوں   O
کے      O
مابین   O
ای      O
حکمرانی O
اقدامات O
کی      O
تاثیر   O
کا      O
جائزہ   O
لیا     O
جا      O
سکے     O
۔       O
```

NIMI, the other one, only occurs in this sentence in the Urdu test data:

```
نرمل    B-NIMI
بھارت   I-NIMI
ابھیان  I-NIMI
کے      O
ذریعہ   O
نرمل    O
گرام    O
پنچایتوں        O
کی      O
تشکیل   O
کا      O
کام     O
لیا     O
گیا     O
اور     O
اس      O
کے      O
لئے     O
سازگار  O
ماحول   O
بنانے   O
کی      O
کوشش    O
کی      O
گئی     O
۔       O
```

**Comment by Pruthwik** [comment]
Thanks for the suggestion. We will look at the data and if needed will update the tags. NIMI was an an error, so we removed it. Similarly, NET looks like an error at first look, we will check it again. NEU was an initial tag that we added in the tagset. But after deliberations, we removed it from the tagset scheme.

**Comment by AngledLuffa** [comment]
Sounds great.  Thanks for the fast response!
